src/libcart/.gitrepo

```diff
@@ -0,0 +1,12 @@
+; DO NOT EDIT (unless you know what you are doing)
+;
+; This subdirectory is a git "subrepo", and this file is maintained by the
+; git-subrepo command. See https://github.com/ingydotnet/git-subrepo#readme
+;
+[subrepo]
+	remote = https://github.com/devwizard64/libcart
+	branch = main
+	commit = 948358b8320f49e75b4a9e3b4e792ccaec866e0b
+	parent = 543d7add024fea312a8991b29a029a0ac0ed7bef
+	method = merge
+	cmdver = 0.4.6
```

Comment:
What's the advantage of subrepos over submodules exactly, and is it necessary here? Seems like subrepos aren't an actual part of git, so I'm hesitant to use this unless submodules/subtrees are no good here.

Reply:
submodules are an extra command for the end user which is Bad™ (think changing the required baserom name bad)

maintaining the subrepo is on the contributors with zero work for the end user

Reply:
It is? Cloning a repo with a submodule doesn't also clone the submodule?

Reply:
an end user would have to either run `git clone --recursive` or `git submodule init --update` after a clone to grab those submodules

Reply:
Welp...that sucks. What about with subtrees?

Reply:
those work but i found them harder to maintain (s2dex text engine used to be a subtree here but i just made it part of the main repo eventually)

Reply:
Well, whatever then. Could you update the libpl PR to use subrepos then since it sounds like submodules are bad